libs/SalesforceOAuth/SalesforceOAuthUnitTests/SalesforceOAuthUnitTests.m

```diff
@@ -254,7 +255,7 @@ - (void)testTokenEncryptionDecryption
 {
     NSString *accessToken = @"gimmeAccess!";
     NSString *refreshToken = @"IWannaRefresh!";
-    SFOAuthCredentials *credentials = [[SFOAuthCredentials alloc] initWithIdentifier:kIdentifier clientId:kClientId encrypted:YES];
+    SFOAuthKeychainCredentials *credentials = [[SFOAuthKeychainCredentials alloc] initWithIdentifier:kIdentifier clientId:kClientId encrypted:YES];
```

Comment:
The methods we use in here are not part of the parent class. I was seeing a lot of build errors because of this. We could move the methods to the parent class, but they are specific to the behavior of the sub class.